packages/tools/devtools/devtools-view/package.json

```diff
@@ -19,7 +19,7 @@
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
 		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
-		"clean": "rimraf _api-extractor-temp coverage dist nyc *.tsbuildinfo *.build.log",
+		"clean": "rimraf \"_api-extractor-temp\" \"coverage\" \"dist\" \"nyc\" \"*.tsbuildinfo\" \"*.build.log\" --glob",
```

Comment:
Note: this script was previously broken - this makes it functional again.

Reply:
Broken in Windows only, I think? Related, but hasn't been merged: https://github.com/microsoft/FluidFramework/pull/15852.

Nit: if Windows doesn't complain with single quotes instead of double, we could also use those to avoid the need for escaping? The behavior in Unix should be the same at least with the patterns we have here ([reference](https://phoenixnap.com/kb/bash-single-vs-double-quotes)).